optimal_guide_finder/guide_strength_calculator.py

```diff
@@ -3,6 +3,7 @@
 from multiprocessing import Process, Queue, Pool
 import numpy as np
 import pandas as pd
+from tqdm import tqdm
```

Comment:
I think the loop on line 33 could be wrapped in this, will probably need this since it's the most time consuming step